packages/@okta/i18n/src/properties/login.properties

```diff
@@ -723,6 +723,7 @@ oie.password.enroll.title = Set up password
 oie.password.passwordLabel = Enter password
 oie.password.confirmPasswordLabel = Re-enter password
 oie.password.reset.verification = Verify with one of the following authenticators to reset your password.
+oie.password.reset.username.or.email.placeholder = Username or email
```

Comment:
I would use this instead 

https://github.com/okta/okta-signin-widget/blob/5054c0687821c7caa7eb64f0465643f2996d2368/packages/@okta/i18n/src/properties/login.properties#L525

Reply:
@vincentvoong-okta is that ok?

Reply:
@gowthamidommety-okta I don't have a strong preference either way. @magizh-okta do you know if there was a reason `Email` is before `username` in the above example?

Reply:
Talked to @vincentvoong-okta offline. This change is preferred in case customers already have this string customized.